File: Java/Baju.java
// kelas turunan dari Aksesoris: Baju
public class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    // konstruktor
    public Baju() {}

    public Baju(int id, String namaProduk, double harga, int stok, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(id, namaProduk, harga, stok, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    // getter
    public String getUntuk() { return untuk; }
    public String getSize() { return size; }
    public String getMerk() { return merk; }

    // menampilkan data dalam bentuk tabel
    public void tampilkan() {
        System.out.printf("%-5d %-15s %-10.2f %-5d %-10s %-10s %-10s %-10s %-5s %-10s\n",
                          id, namaProduk, harga, stok, jenis, bahan, warna, untuk, size, merk);
    }
}

File: CPP/Baju.cpp
#include "Aksesoris.cpp"

// kelas turunan dari Aksesoris: Baju
class Baju : public Aksesoris {
private:
    string untuk;
    string size;
    string merk;

public:
    // konstruktor
    Baju() {}
    Baju(int id, string nama_produk, double harga, int stok, string jenis, string bahan, string warna, string untuk, string size, string merk)
        : Aksesoris(id, nama_produk, harga, stok, jenis, bahan, warna) {
        this->untuk = untuk;
        this->size = size;
        this->merk = merk;
    }

    // getter
    string getUntuk() { return untuk; }
    string getSize() { return size; }
    string getMerk() { return merk; }

    // setter
    void setUntuk(string untuk) { this->untuk = untuk; }
    void setSize(string size) { this->size = size; }
    void setMerk(string merk) { this->merk = merk; }

    // menampilkan data dalam bentuk tabel
    void tampilkan() {
        cout << id << "\t" << nama_produk << "\t" << harga << "\t" << stok << "\t"
             << jenis << "\t" << bahan << "\t" << warna << "\t"
             << untuk << "\t" << size << "\t" << merk << endl;
    }
};
